LocalPackages/DataBrokerProtection/Package.swift

```diff
@@ -44,7 +44,7 @@ let package = Package(
                 .product(name: "Configuration", package: "BrowserServicesKit"),
                 .product(name: "Persistence", package: "BrowserServicesKit"),
             ],
-            resources: [.process("Resources")],
+            resources: [.copy("Resources")],
```

Comment:
An unfortunate sacrifice to preserve the file structure. Although I don't think it will actually affect anything for now since we don't have e.g. any images in here

Reply:
Yeah exactly, I think it’s fine for JSON 👍🏼